harness/determined/deploy/aws/templates/efs.yaml

```diff
@@ -730,6 +730,13 @@ Resources:
                   - cloudwatch:PutMetricData
                 Effect: Allow
                 Resource: "*"
+        - PolicyName: master-s3-policy
+          PolicyDocument:
+            Version: 2012-10-17
```

Comment:
will we ever have to bump this version or is it pretty much good?

Reply:
This looks to still be the most recent version for policy document.